### DISCUSSION TITLE
Error executing query: javax.net.ssl.SSLHandshakeException: Remote host terminated the handshake

### DISCUSSION THREAD

**Discussion by rafagfe** [starburstdata/metabase-driver#125] [Q&A]
Hi guys,

I'm using the driver 'starburst-4.1.0.metabase-driver.jar' plugin to access the 'starburst' database and connect the Trino + Metabase, however, when making the connection without SSL, I receive the message in the metabase UI:
'TLS/SSL is required for authentication with username and password'.

When I select the 'ssl' option and try to connect, I receive the message:
"Error executing query: [javax.net](http://javax.net/).ssl.SSLHandshakeException: Remote host terminated the handshake'.

I don't want to use SSL/TLS. Does anyone know how to fix it?

in the 'config.properties' file I have these configs:

coordinator=true
node-scheduler.include-coordinator=true
http-server.http.enabled=true
http-server.http.port=8080
query.max-memory=5GB
query.max-memory-per-node=1GB
discovery-server.enabled=true
discovery.uri=http://localhost:8080/

in the 'etc/catalog/postgresql-rds.properties' i have these configs:

connector.name=postgresql
connection-url=jdbc:postgresql://dbtest11.test.com.br:5432/test
connection-user=xxxx
connection-password=xxxx

In 'trino-cli' I connect normally. look:

(venv) rgf@rafaelgf:~/projetos/docker/trino$ docker exec -it trino trino --server localhost:8080 --catalog postgresql-rds --schema public
trino:public> show catalogs;
     Catalog

 postgresql-rds
 postgresql-local
 system
(3 rows)
Query 20240418_174412_00002_wbwcd, FINISHED, 1 node
Splits: 11 total, 11 done (100.00%)
0.19 [0 rows, 0B] [0 rows/s, 0B/s]
trino:public> show tables;
                      Table

 _disk_space_check_server_airflow
 _table_size_data
 _tables_projects (editado)

**Comment by lpoulain** [comment]
The error says it all: Trino requires SSL if you want to use a password. Put an empty password in your database parameters and this should work.
